backend/jobs/eventsPusher.ts

```diff
@@ -39,15 +39,15 @@ export default class EventsPusher {
       logger.info(`EventsPusher: Finished. No events to push.`)
       return
     }
-    const firstRow = parseInt(events[0][0], 10)
-    const range = `A${firstRow + 1}`
```

Comment:
no longer using the eventId as a the row to start adding new data from. Now will always add from A2